conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
> If there won't be any new environments in the future along this naming convention, then it makes sense to stop at "6" - but there's already 3, 5, and 6 - the cowpath is pretty paved imo.

There is only an `es6` environment, these others you mention don't exist. We do support those versions for the ecmaVersion option. 

> What happens when es5 is false and es6 is true? Isn't it the same thing, where es6 subsumes es5, so setting es5 to false has no effect? (like setting ecmaVersion, rather than like setting env flags)

Again, es5 environment doesn't exist.